lib/rules/use-colors/index.js

```diff
@@ -0,0 +1,80 @@
+/*
+ Copyright 2020 Skyscanner
+ Licensed under the Apache License, Version 2.0 (the "License");
+ you may not use this file except in compliance with the License.
+ You may obtain a copy of the License at
+
+ http://www.apache.org/licenses/LICENSE-2.0
+
+ Unless required by applicable law or agreed to in writing, software
+ distributed under the License is distributed on an "AS IS" BASIS,
+ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ See the License for the specific language governing permissions and
+ limitations under the License.
+ */
+
+const stylelint = require('stylelint');
+const _ = require('lodash');
+const tinycolor = require('tinycolor2');
+
+const { tokensByCategory } = require('../../utils/token-utils');
+
+const COLOR_ALLOWLIST = ['transparent'];
```

Comment:
```suggestion
const COLOR_ALLOWLIST = ['transparent'];
```

Nice use of allowlist too!

Reply:
Using inclusive langauge 😄